build.sh

```diff
@@ -268,13 +268,8 @@ if  [ ${BUILD_ALL_GPU_ARCH} -eq 1 ]; then
     echo "Building for *ALL* supported GPU architectures..."
 else
     if [ ${BUILD_CI_ONLY} -eq 1 ]; then
-        if [[ ${CUDA_VERSION} == 11* ]]; then
-            CUOPT_CMAKE_CUDA_ARCHITECTURES="70-real;80"
-            echo "Building for Volta and Ampere architectures..."
-        else
-            CUOPT_CMAKE_CUDA_ARCHITECTURES="RAPIDS"
-            echo "Building for Volta, Ampere and Hopper architectures..."
-        fi
+        CUOPT_CMAKE_CUDA_ARCHITECTURES="RAPIDS"
+        echo "Building for Volta, Ampere and Hopper architectures..."
```

Comment:
This comment is not accurate, because the definition of "RAPIDS" arches changes over time. Currently that includes this set of arches: https://github.com/rapidsai/rapids-cmake/blob/4eb862d7eb00ee3bfe79c001fca79c8685c2dadb/rapids-cmake/cuda/set_architectures.cmake#L59

I would change this:
```suggestion
        CUOPT_CMAKE_CUDA_ARCHITECTURES="RAPIDS"
        echo "Building for RAPIDS supported architectures..."
```